dallinger/data.py

```diff
@@ -281,12 +290,23 @@ def export_db_uri(id, db_uri, local, scrub_pii):
                 f" - bucket name: {bucket.name}\n"
                 f" - S3 console URL: {s3_console_url}"
             )
-        except AttributeError:
-            raise S3BucketUnavailable("Could not find an S3 bucket!")
+        else:
+            print("✘ Could not find an S3 bucket!")
```

Comment:
It looks like this error message will be printed if no AWS credentials are found. This doesn't seem like a sensible message for that situation.

Reply:
OK, I think I finally got what solution you were looking for :+1: